fvm/transactionVerifier.go

```diff
@@ -50,9 +57,11 @@ func (v *TransactionSignatureVerifier) verifyTransactionSignatures(
 	tx := proc.Transaction
 	accounts := state.NewAccounts(sth)
 	if tx.Payer == flow.EmptyAddress {
-		return &MissingPayerError{}
+		err := errors.NewInvalidAddressError("payer address is invalid", tx.Payer)
```

Comment:
A Missing payer or a missing proposer has separate consequences than a missing authoriser. Missing authoriser means the fees can still be deducted. Missing payer/proposer means they cant be.

Not sure about this though. It might be easier to just say that the collection node should reject transaction where ANY signature is missing.

Reply:
To be honest these checks are currently happening at collection time, so we might even remove them here.